core/src/main/java/org/elasticsearch/common/util/CancellableThreads.java

```diff
@@ -92,6 +94,7 @@ public void execute(Interruptable interruptable) {
      *
      * @param interruptable code to run
      */
+    @SuppressForbidden(reason = "Use Thread.interrupted() to implement this class' interruption policy (forbid external interrupts).")
```

Comment:
> I am not happy at all that @SuppressForbidden takes an all or nothing approach

There is an open issue for just that:
https://github.com/policeman-tools/forbidden-apis/issues/96